AdyenActions/Components/QRCode/QRCodeComponent.swift

```diff
@@ -157,9 +157,23 @@ public final class QRCodeComponent: ActionComponent, Cancellable {
     private func createViewController(with action: QRCodeAction) -> UIViewController {
         let viewController = QRCodeViewController(viewModel: createModel(with: action))
         viewController.qrCodeView.delegate = self
+        viewController.navigationItem.rightBarButtonItem = createRightNavigationBarItem()
         return viewController
     }
     
+    private func createRightNavigationBarItem() -> UIBarButtonItem {
```

Comment:
createRightNavigationBarItem and onTrailingButtonTap are duplicated, maybe we can move it to a common protocol